### PR TITLE
Added comments to util.go #774

### PR DESCRIPTION
**Pull request by Ankitkurani1997**
**What this PR does / why we need it**: 
   Refer to issue openebs/openebs#774

**Which issue this PR fixes** *(optional, in fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: 
  fixes #774

**Special notes for your reviewer**:

### CODE REVIEW

File: orchprovider/k8s/v1/util.go [hunk 1]
```diff
@@ -370,7 +370,7 @@ func (p *VolumeMarkerBuilder) AddVolumeCapacity(deploy k8sApisExtnsBeta1.Deploym
 	_ = p.Add(string(v1.CapacityVK), capacity)
 }
 
-//
+// AddIQN adds an iqn
```
Comment (utkarshmani1997) [inline orchprovider/k8s/v1/util.go]: change this too accordingly

File: orchprovider/k8s/v1/util.go [hunk 2]
```diff
@@ -383,7 +383,7 @@ func (p *VolumeMarkerBuilder) AddIQN(volumeName string) {
 	_ = p.Add(string(v1.JivaIQNVK), iqn)
 }
 
-//
+// AddControllerClusterIP adds controllercluster ip
```
Comment (utkarshmani1997) [inline orchprovider/k8s/v1/util.go]: Change this one too

File: orchprovider/k8s/v1/util.go [hunk 3]
```diff
@@ -457,7 +457,7 @@ type LabelK8sObject struct {
 	// to the targetted K8s Object
 	LabelValue string
 }
-
+// NewLabelK8sObject returns a LabelK8sObject
```
Comment (utkarshmani1997) [inline orchprovider/k8s/v1/util.go]: change this to "returns an instance of LabelK8sObject whcih contains the key and value of the k8s labels"

**Comment by Ankitkurani1997** [comment]
Can you please let us know what are changes expected from us.

**Comment by Ankitkurani1997** [comment]
@kmova can you please let us know to what extent are the details in the comments expected.

**Comment by kmova** [comment]
Hi @Ankitkurani1997  - First of all.. thank you for contributing to this project.

About this particular PR - The goal is to make sure we are adding comments that actually help us understand this code. Take the example of OpenEBSImage  -- the comments should be able to provide answers to what this function is and where it is used and why was this function even required. The comments should be more descriptive than the name itself. 

Refer: https://golang.org/doc/effective_go.html?#introduction

**Comment by kmova** [comment]
The PR has been inactive for a long time. Please reopen this PR by incorporating the review comments, if you would like to continue to fix this issue.

**Pull request closed without merging**